gpkit/tests/t_constraints.py

```diff
@@ -49,16 +47,16 @@ def test_oper_overload(self):
         x = Variable('x')
         y = Variable('y')
         c = (y >= 1 + x**2)
-        self.assertEqual(c, 1/y + x**2/y)
+        self.assertEqual(c.posylt1_rep, 1/y + x**2/y)
```

Comment:
`as_posyslt1` is a required interface for a constraint, `posylt1_rep` is an attribute of this particular implementation. This test should probably be `as_posyslt1()`, will change.